openshift/tests-extension/.openshift-tests-extension/openshift_payload_olmv1.json

```diff
@@ -18,5 +18,35 @@
     "source": "openshift:payload:olmv1",
     "lifecycle": "blocking",
     "environmentSelector": {}
+  },
+  {
+    "name": "[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 operator installation should install a cluster extension",
```

Comment:
See that matches with the name: https://github.com/openshift/origin/blob/05eeb94cf14384be99323b2cbd00ab61b1b1c47c/test/extended/util/annotate/generated/zz_generated.annotations.go#L2000